### PR TITLE
zh-cn: sync the translation of runtime.sendMessage()

### PR DESCRIPTION
**Pull request by yin1999**
### Description

sync the translation of runtime.sendMessage()

### Related issues and pull requests

Part of: #14272

### CODE REVIEW

File: files/zh-cn/mozilla/add-ons/webextensions/api/runtime/sendmessage/index.md [hunk 1]
```diff
@@ -1,108 +1,119 @@
...
 
-> **备注：** 在 Firefox 55 之前，只给出 2 个参数时，规则会有所不同：
-> Under the old rules, if the first argument was a string, it was treated as the `extensionId`, with the message as the second argument. This meant that if you called `sendMessage()` with arguments like `("my-message", {})`, then it would send an empty message to the extension identified by "my-message". Under the new rules, with these arguments you would send the message "my-message" internally, with an empty options object.
+注意，在 Firefox 55 之前，2 个参数的情况下的规则是不同的。在旧规则下，如果第一个参数是字符串，则将其视为 `extensionId`，第二个参数作为消息。这意味着如果你使用 `("my-message", {})` 这样的参数调用 `sendMessage()`，那么它将向标识为“my-message”的扩展发送空消息。根据新规则，使用这样的参数，将会是在带有一个空的 options 对象的情况下，在内部发送消息“my-message”。
 
-### Return value
+### 返回值
 
-返回一个 [`Promise`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise)。若接收方响应，Promise 将会变为 fulfilled 并且返回接收方响应的 JSON 对象（数字、字符串、数组、true/false 都是合法的 JSON 对象）。否则，Promise 会变为 fulfilled 但是不返回任何参数。如果发生了连接错误，Promise 将会变为 rejected 并返回一个错误消息。
+一个 [`Promise`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise)。若接收方发送来响应，Promise 将会兑现为该响应。否则，它将不兑现任何参数。如果连接到扩展时发生了错误，则 Promise 将以一个错误消息拒绝。
```
Comment (jasonren0403) [inline files/zh-cn/mozilla/add-ons/webextensions/api/runtime/sendmessage/index.md:71]:
```suggestion
一个 [`Promise`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise)。若接收方发送了响应，Promise 将会兑现为该响应。否则，它将不带任何参数兑现。如果连接到扩展时发生了错误，则 Promise 将以一个错误消息拒绝。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/runtime/sendmessage/index.md [hunk 2]
```diff
@@ -1,108 +1,119 @@
...
-The corresponding background script looks like this:
+对应的后台脚本如下所示：
 
 ```js
 // background-script.js
-
 function handleMessage(request, sender, sendResponse) {
-  console.log("Message from the content script: " + request.greeting);
-  sendResponse({ response: "Response from background script" });
+  console.log(`A content script sent a message: ${request.greeting}`);
```
Comment (jasonren0403) [inline files/zh-cn/mozilla/add-ons/webextensions/api/runtime/sendmessage/index.md:107]:
```suggestion
  console.log(`内容脚本发送了一条消息：${request.greeting}`);
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/runtime/sendmessage/index.md [hunk 3]
```diff
@@ -1,108 +1,119 @@
...
+  sendResponse({ response: "来自后台脚本的响应" });
 }
 
 browser.runtime.onMessage.addListener(handleMessage);
 ```
```
Comment (jasonren0403) [inline files/zh-cn/mozilla/add-ons/webextensions/api/runtime/sendmessage/index.md:113]: https://github.com/mdn/content/blob/925b2bd8beeae6ce8237863637bcd28ccbb8d47f/files/en-us/mozilla/add-ons/webextensions/api/runtime/sendmessage/index.md?plain=1#L114

这里少了原文档的114-115行

**Review by jasonren0403 — COMMENTED** [review]
一些建议

File: files/zh-cn/mozilla/add-ons/webextensions/api/runtime/sendmessage/index.md (hunk 3 above)
Comment (yin1999) [reply files/zh-cn/mozilla/add-ons/webextensions/api/runtime/sendmessage/index.md:113]:
```suggestion

> **备注：** 对于 Firefox 附加组件，建议使用返回 [`Promise`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise) 的方法，而不是使用 `sendResponse()`。

```

**Review by jasonren0403 — APPROVED** [review]

**Pull request merged**